framework/material/Form.kt

```diff
@@ -204,13 +210,13 @@ object Form {
     }
 
     @Composable
-    fun RowSpacer() {
-        Spacer(modifier = Modifier.width(INNER_SPACING))
+    fun RowSpacer(multiplier: Int = 1) {
```

Comment:
Let's use `DIALOG_SPACING` instead of a double-height `ColumnSpacer` and double-width `RowSpacer`. Just keeps the spacers cleaner and more consistent